cromwellApiClient/src/main/scala/cromwell/api/CromwellClient.scala

```diff
@@ -92,18 +92,27 @@ class CromwellClient(val cromwellUrl: URL, val apiVersion: String)(implicit acto
   def metadata(workflowId: WorkflowId)(implicit ec: ExecutionContext): Future[WorkflowMetadata] = getRequest[String](metadataEndpoint(workflowId)) map WorkflowMetadata
   def backends(implicit ec: ExecutionContext): Future[CromwellBackends] = getRequest[CromwellBackends](backendsEndpoint)
   def version(implicit ec: ExecutionContext): Future[CromwellVersion] = getRequest[CromwellVersion](versionEndpoint)
+  
+  private [api] def executeRequest(request: HttpRequest) = Http().singleRequest(request)
 
   /**
     *
     * @tparam A The type of response expected. Must be supported by an implicit unmarshaller from ResponseEntity.
     */
   private def makeRequest[A](request: HttpRequest)(implicit um: Unmarshaller[ResponseEntity, A], ec: ExecutionContext): Future[A] = for {
-    response <- Http().singleRequest(request)
+    response <- executeRequest(request)
     decoded <- Future.fromTry(decodeResponse(response))
     entity <- Future.fromTry(decoded.toEntity)
-    unmarshalled <- entity.to[A]
+    unmarshalled <- unmarshall(response, entity)(um, ec)
```

Comment:
actually there's an implicit conflict which is why I had to explicitly pass them